src/fiatExchanges/quotes/ExternalQuote.ts

```diff
@@ -92,15 +93,28 @@ export default class ExternalQuote extends NormalizedQuote {
     return null
   }
 
+  getReqsSubtitle(): string | null {
+    return !isSimplexQuote(this.quote) &&
+      this.getPaymentMethod() === PaymentMethod.Airtime &&
+      this.quote.extraReqs
+      ? this.quote.extraReqs.mobileCarrier ?? ''
+      : this.getKycInfo()
```

Comment:
```suggestion
    return this.getPaymentMethod() === PaymentMethod.Airtime &&
      this.quote.extraReqs?.mobileCarrier ??
      : this.getKycInfo()
```
if extraReqs is truthy but mobileCarrier is not, seems like it should show kyc info and not an empty string

also, the simplex part doesn't seem relevant

Reply:
one more thing-- we need it to say "MTN required", not just "MTN". The string should also be localized. I slacked you some information on how to accomplish that

Reply:
I think the Simplex part is needed, I tried getting rid of it and `this.quote.extraReqs?` gives an error saying extraReqs does not exist in SimplexQuote. I think I could add an optional field to SimplexQuote for extraReqs which never gets used but otherwise I think I need this check.

Reply:
ok I didn't realize the simplex check was a type guard